### PR TITLE
Add Events 6 and 7

### PR DESCRIPTION
**Pull request by loicm**

### CODE REVIEW

File: data/events.json [hunk 1]
```diff
@@ -36,5 +36,23 @@
             {"speaker": "Mehdi et Loïc", "subject": "Présentation Web en Vert"},
             {"speaker": "Mehdi Lahmam", "subject": "Remote"}
         ]
+    },
+    {
+        "title": "Web en Vert #6 - Talks",
+        "date": "2014-12-18",
+        "localisation": "Axome",
```
Comment (mehlah) [inline data/events.json]: Should be `location`

File: data/events.json [hunk 2]
```diff
@@ -36,5 +36,23 @@
...
+        "description": "De nouveau accueillis par une entreprise locale. Web en Vert commence à se faire connaitre et à plaire aux acteurs locaux. De nouvelles têtes et des débats très enrichissants pour finir l'année !",
+        "talks": [
+            "speaker": "Loïc Mathaud", "subject": "Présentation de l'IndieWeb"
```
Comment (mehlah) [inline data/events.json]: missing `{` here. We expect an array of objects.

**Comment by mehlah** [comment]
I left a couple of comments!
:+1: from me

**Comment by loicm** [comment]
Is it ok now?

**Comment by mehlah** [comment]
What about https://github.com/craftsmen/web-en-vert/pull/10#discussion_r24918341?

**Comment by loicm** [comment]
Arg… I didn't see this comment, sorry. I fixed it.

**Comment by mehlah** [comment]
LGTM :+1:

**Comment by loicm** [comment]
Better history with a rebase.

**Pull request merged**